std/math.d

```diff
@@ -4531,6 +4545,10 @@ public:
      /// A machine NaN was generated. (example: x = real.infinity * 0.0; )
      @property bool invalid() { return (flags & INVALID_MASK) != 0; }
 
+     /** A subnormal number was used in a calculation.
+     (example: x = real.min_normal / 2 + 1) */
+     @property bool subnormal() const { return (flags & SUBNORMAL_MASK) != 0; }
```

Comment:
> The majority of targets don't have an exception for denormals, I don't think this should be added here.

As far as I see, IeeeFlags only works on x86 and x86-64 at the moment. Everything else errors out with "not yet supported". x86(-64) are the ones that have this, right? Would it be ok if this was `version`ed away from the others?